account_reconcile_model_multicompany_propagate/models/account_reconcile_model.py

```diff
@@ -139,7 +140,7 @@ def _get_account_map_propagate(self, companies):
                 .with_company(com_id)
                 .search(
                     [
-                        ("company_ids", "in", companies.ids),
+                        ("company_ids", "in", com_id),
```

Comment:
I don't know about the logic, but this is going to fail if not in an enumerator object:
```suggestion
                        ("company_ids", "in", [com_id]),